notebooks/useful_notebooks/01_find_and_release.ipynb

```diff
@@ -221,25 +221,25 @@
     "    print('Number of experiments is not same as experiments associated with sets')\n",
     "    print('# of exps: {}. # of exps on sets: {}'.format(len(all_exps), len(all_exps_on_sets)))\n",
     "\n",
-    "hela_exps = []  # a list of exp['uuid'] with biosource from HeLa individual\n",
-    "hela_exps_unsure = []\n",
+    "restricted_exps = []  # a list of exp['uuid'] with biosource from restricted individual\n",
+    "restricted_exps_unsure = []\n",
     "for exp_type in exp_names:\n",
     "    for exp in store[exp_type]:\n",
-    "        biosource_is_hela = 0\n",
+    "        biosource_is_restricted = 0\n",
     "        for bs in exp['biosample']['biosource']:\n",
-    "            if bs.get('individual') and bs['individual'].get('display_title') == '4DNINEL8T2GK':  # the HeLa individual\n",
-    "                biosource_is_hela += 1\n",
-    "        if biosource_is_hela == len(exp['biosample']['biosource']):  # all biosources are HeLa: exp is HeLa\n",
-    "            hela_exps.append(exp['uuid'])\n",
-    "        elif biosource_is_hela > 0:  # some but not all biosources are HeLa: unsure\n",
-    "            hela_exps_unsure.append(exp['uuid'])\n",
-    "if hela_exps_unsure and change_level >= 9:\n",
-    "    release_hela = input('Experiments with multiple Biosources found, some of which are HeLa. ' +\n",
+    "            if bs.get('individual') and bs['individual'].get('protected_data') == 'true':\n",
```

Comment:
Are we sure in the context of the notebook and based on the boolean type specified in the schema that the value is actually returned as 'true'?  I thought that this should likely be an actual Boolean test and not looking for the string but am not sure.